linkerd/app/outbound/src/target.rs

```diff
@@ -45,6 +42,11 @@ pub struct Endpoint<P> {
     pub protocol: P,
 }
 
+#[derive(Copy, Clone)]
+pub struct EndpointFromMetadata {
+    pub identity_disabled: bool,
+}
+
```

Comment:
is there a reason this moved around? not a big deal.

Reply:
just putting near `Endpoint`